recipes/myjive/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "myjive" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/myjive-{{ version }}.tar.gz
+  sha256: 91ec0f08d9ff2065f84c48805c84a42bc3193812a66fc1f54c4e24aa43d191df
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - matplotlib-base ==3.5.2
+    - numba ==0.56.4
+    - numpy ==1.23.5
+    - scikit-sparse ==0.4.12
+    - scipy ==1.8.1
```

Comment:
Whilst these specific dependencies are fine, this will make the installation very difficult in the future. Maybe consider relaxing these versions at some point.